pkg/logs/status/warnings.go

```diff
@@ -0,0 +1,66 @@
+// Unless explicitly stated otherwise all files in this repository are licensed
+// under the Apache License Version 2.0.
+// This product includes software developed at Datadog (https://www.datadoghq.com/).
+// Copyright 2018 Datadog, Inc.
+
+package status
+
+import (
+	"fmt"
+
+	"github.com/DataDog/datadog-agent/pkg/logs/config"
+)
+
+// Warning types
+const (
+	OpenFilesLimitReachedType = "open_files_limit_reached"
+)
+
+// warnings collect the status of warnings
+var (
+	Warnings = NewWarnings()
+)
+
+// WarningsStruct holds a warning message
+type WarningsStruct struct {
+	RaisedWarnings map[string]bool
+	Wildcards      map[string]int
+}
+
+// NewWarnings initialize Warnings with the default values
+func NewWarnings() *WarningsStruct {
+	raisedWarnings := make(map[string]bool)
+	raisedWarnings[OpenFilesLimitReachedType] = false
+	wildcards := make(map[string]int)
+
+	return &WarningsStruct{
+		RaisedWarnings: raisedWarnings,
+		Wildcards:      wildcards,
+	}
+}
+
+// GenerateMessages returns the list of warning raised
+func (w WarningsStruct) GenerateMessages() []string {
+	var messages []string
+	for warningType, isWarningRaised := range w.RaisedWarnings {
+		if isWarningRaised {
+			switch warningType {
+			case OpenFilesLimitReachedType:
+				messages = append(
+					messages,
+					fmt.Sprintf(
+						"The default limit on the maximum number of files in use (%d) has been reached. Increase this limit (thanks to the attribute logs_config.open_files_limit in datadog.yaml) or decrease the number of tailed file.",
```

Comment:
We can still reach the limit if we override the files limit -> it's not always the `default limit` that has been reached

Reply:
Yes, the number displayed is the actual configured number, I will update the wording